policy/cnspec_policy.proto

```diff
@@ -554,12 +554,16 @@ message ReportingJob {
 
   enum Type {
     UNSPECIFIED = 0;
-    CHECK = 1;
-    DATA_QUERY = 2;
     CONTROL = 3;
     POLICY = 4;
     FRAMEWORK = 5;
     RISK_FACTOR = 6;
+
+    // DO NOT USE CHECK OR DATA_QUERY, THEY ARE DEPRECATED
```

Comment:
What should be used?
Or what is the equivalent to `CONTROL` for a policy?

Reply:
i've left it as UNSPECIFIED. it's not used by anything